* Don't compress man pages
* Don't override compiler optimization flags
A similar patch has been already forwarded to upstream by Debian maintainers
--- zita-ajbridge-0.8.4.orig/source/Makefile	2020-04-05 23:36:32.000000000 +0300
+++ zita-ajbridge-0.8.4/source/Makefile	2025-06-26 07:33:03.834940444 +0300
@@ -24,10 +24,10 @@
 
 VERSION = 0.8.4
 CPPFLAGS +=  -MMD -MP -DVERSION=\"$(VERSION)\"
-CXXFLAGS += -O2 -Wall
+CXXFLAGS += -Wall
 
 
-all:	zita-a2j zita-j2a zita-ajbridge.1.gz zita-a2j.1.gz zita-j2a.1.gz 
+all:	zita-a2j zita-j2a
 
 
 ZITA-A2J_O = zita-a2j.o alsathread.o jackclient.o pxthread.o lfqueue.o
@@ -63,9 +63,9 @@
 	install -d $(DESTDIR)$(MANDIR)
 	install -m 755 zita-a2j $(DESTDIR)$(BINDIR)
 	install -m 755 zita-j2a $(DESTDIR)$(BINDIR)
-	install -m 644 zita-ajbridge.1.gz $(DESTDIR)$(MANDIR)
-	install -m 644 zita-a2j.1.gz $(DESTDIR)$(MANDIR)
-	install -m 644 zita-j2a.1.gz $(DESTDIR)$(MANDIR)
+	install -m 644 zita-ajbridge.1 $(DESTDIR)$(MANDIR)
+	install -m 644 zita-a2j.1 $(DESTDIR)$(MANDIR)
+	install -m 644 zita-j2a.1 $(DESTDIR)$(MANDIR)
 
 
 uninstall:
